libcloud/test/backup/test_dimensiondata.py

```diff
@@ -130,18 +130,18 @@ def test_ex_get_backup_details_for_target(self):
         self.assertEqual(client.type.type, 'FA.Linux')
         self.assertEqual(client.running_job.progress, 5)
         self.assertTrue(isinstance(client.running_job, BackupTargetJob))
-        self.assertEqual(len(client.alert.notify_list), 2)
-        self.assertTrue(isinstance(client.alert.notify_list, list))
-
-    def test_ex_get_backup_details_for_target_NOBACKUP(self):
-        target = self.driver.list_targets()[0].address
-        DimensionDataMockHttp.type = 'NOBACKUP'
-        response = self.driver.ex_get_backup_details_for_target(target)
-        self.assertTrue(response is None)
-
-    def test_ex_cancel_target_job(self):
-        target = self.driver.list_targets()[0]
-        response = self.driver.ex_get_backup_details_for_target(target)
+        self.assertEqual(len(client.alert.notify_list), 2)
+        self.assertTrue(isinstance(client.alert.notify_list, list))
+
+    def test_ex_get_backup_details_for_target_NOBACKUP(self):
+        target = self.driver.list_targets()[0].address
+        DimensionDataMockHttp.type = 'NOBACKUP'
+        response = self.driver.ex_get_backup_details_for_target(target)
+        self.assertTrue(response is None)
+
+    def test_ex_cancel_target_job(self):
+        target = self.driver.list_targets()[0]
+        response = self.driver.ex_get_backup_details_for_target(target)

```

Comment:
I think this is highlighting a line break change. can you check in your editor please that line endings are normalised.